Modules/DRSRule.md

```diff
@@ -0,0 +1,18 @@
+# Name: DRSRule
+
+## Description:
+The DRSRule module allows you to work with all types of vSphere DRS rules. The module provides support for VM and VMHost groups. And, it works with affinity/anti-affinity VM rules and VM to VMHost rules.
+
+| Works with | Author(s)  | Project URL(s) | Last Update   | Tags |
+|------------|------------|----------------|---------------|------|
+| vSphere 5+ | Matt Boren (<a href="http://www.vnugglets.com" target="_blank">b</a> \| <a href="https://twitter.com/mtboren" target="_blank">t</a>), Luc Dekens (<a href="http://www.lucd.info" target="_blank">b</a> \| <a href="https://twitter.com/LucD22" target="_blank">t</a>) | [GitHub](https://github.com/PowerCLIGoodies/DRSRule) | May 2016 | vSphere, Cluster, DRS, Rules, Groups |
+
+### Shared blog posts:
+Matt Boren (<a href="http://www.vnugglets.com" target="_blank">b</a> \| <a href="https://twitter.com/mtboren" target="_blank">t</a>)
+- [DRSRule – a DRS rules and groups module](http://www.lucd.info/2015/01/22/drsrule-drs-rules-and-groups-module/) @ 2015-01-22
```

Comment:
I think the blog posts need to be switch. This one is yours and the other under your name is from Matt.
Everything eles seems ok! 😄